app/controllers/fhir/r4/api_controller.rb

```diff
@@ -424,6 +424,120 @@ def all
     status_ok(bundle) && return
   end
 
+  # Kick off async generation of bulk FHIR data for all accessible patients
+  #
+  # GET /fhir/r4/Patient/$export
+  def bulk_data_export
+    # Require all scopes for all possible resources being returned
+    return if doorkeeper_authorize!(*PATIENT_READ_SCOPES)
+    return if doorkeeper_authorize!(*OBSERVATION_READ_SCOPES)
+    return if doorkeeper_authorize!(*QUESTIONNAIRE_RESPONSE_READ_SCOPES)
+    return if doorkeeper_authorize!(*RELATED_PERSON_READ_SCOPES)
+    return if doorkeeper_authorize!(*IMMUNIZATION_READ_SCOPES)
+    return if doorkeeper_authorize!(*PROVENANCE_READ_SCOPES)
+
+    status_not_acceptable_with_custom_errors(["'Accept' header must have a value of 'application/fhir+json'"]) && return unless accept_header?
```

Comment:
I think `accept_header?` should only allow `json`, `application_json` and `application/fhir+json` if the `_format` param is specified to override the `Accept` header, which is a more general part of the FHIR spec: https://www.hl7.org/fhir/http.html#parameters. It's a bit unclear if that part of the FHIR spec still applies here, but I think it's fair to say yes and be more permissive since it shouldn't cause any issues.

That does mean the message is still slightly off though, it should indicate the values that can be used in the `_format` parameter, or else someone trying to override `Accept` via `_format` (and doing it wrong) will get a confusing error.

Reply:
Improved error message [here](https://github.com/SaraAlert/SaraAlert/pull/987/commits/6977cd314b84a7c5ecfd8d2d2ac200bf5d218e7b#diff-e79e38176ee697a8541cffba23c60d9decaa69382ca5c6e875da033508e5b428R439-R442)